cpp/src/round/round.cu

```diff
@@ -31,6 +33,9 @@ namespace {  // anonymous
 float __device__ generic_round(float f) { return roundf(f); }
 double __device__ generic_round(double d) { return ::round(d); }
 
+float __device__ generic_round_half_even(float f) { return rintf(f); }
```

Comment:
`inline` all of these functions?

Reply:
Compiler automatically inlines these functions.

Reply:
Actually NVCC inlines _everything_ unless you compile with -rdc. But I'm worried about what will happen if we one day turn that on.

Reply:
Didn't realize that was the case with NVCC. Will add.

Reply:
Done.